Integrations/BeyondTrust_Password_Safe/BeyondTrust_Password_Safe.py

```diff
@@ -0,0 +1,316 @@
+            'AccountId': account.get('AccountId'),
+            'AccountName': account.get('AccountName')
+        })
+
+    ec = {'BeyondTrust.Account(val.AccountId === obj.AccountId)': managed_accounts}
+
+    return_outputs(tableToMarkdown('BeyondTrust Managed Accounts', data, removeNull=True), ec, managed_accounts)
+
+
+def get_managed_systems_request():
+
+    suffix_url = '/managedsystems'
+    response = http_request('GET', suffix_url)
+
+    return response
+
+
+def get_managed_systems():
+    """
+    Returns a list of Managed Systems.
+    """
+    data = []
+    managed_systems = get_managed_systems_request()
+    for managed_system in managed_systems:
+        data.append({
+            'ManagedSystemID': managed_system.get('ManagedSystemID'),
+            'ChangeFrequencyDays': managed_system.get('ChangeFrequencyDays'),
+            'AssetID': managed_system.get('AssetID'),
+            'DatabaseID': managed_system.get('DatabaseID'),
+            'DirectoryID': managed_system.get('DirectoryID'),
+            'SystemName': managed_system.get('SystemName'),
+            'PlatformID': managed_system.get('PlatformID'),
+            'Port': managed_system.get('Port')
+        })
+
+    ec = {'BeyondTrust.System(val.ManagedSystemID === obj.ManagedSystemID)': managed_systems}
+
+    return_outputs(tableToMarkdown('BeyondTrust Managed Systems', data, removeNull=True), ec, managed_systems)
+
+
+def create_release_request(data):
+
+    suffix_url = '/requests'
+    response = http_request('POST', suffix_url, data=data)
+
+    return response
+
+
+def create_release():
+    """
+    Creates a new release request.
+    Retrieves the credentials for an approved and active (not expired) credentials release request.
+    """
+    access_type = demisto.args().get('access_type')
+    system_id = demisto.args().get('system_id').encode('utf-8')
+    account_id = demisto.args().get('account_id').encode('utf-8')
+    duration_minutes = demisto.args().get('duration_minutes').encode('utf-8')
+    reason = demisto.args().get('reason')
+    access_policy_schedule_id = demisto.args().get('access_policy_schedule_id')
+    conflict_option = demisto.args().get('conflict_option')
+
+    data = {
+        'SystemId': system_id,
+        'AccountId': account_id,
+        'DurationMinutes': duration_minutes
+    }
+
+    if access_type:
+        data['AccessType'] = access_type
+
+    if reason:
+        data['Reason'] = reason
+
+    if access_policy_schedule_id:
+        data['AccessPolicyScheduleID'] = access_policy_schedule_id
+
+    if conflict_option:
+        data['ConflictOption'] = conflict_option
+
+    request = create_release_request(str(data))
+    request_id = str(request)
+
+    credentials = get_credentials_request(request_id)
+
+    response = {
+        'RequestID': request_id,
+        'Credentials': credentials
+    }
+
+    ec = {'BeyondTrust.Request(val.AccountId === obj.AccountId)':createContext(response)}
```

Comment:
ad white space after
```suggestion
    ec = {'BeyondTrust.Request(val.AccountId === obj.AccountId)': createContext(response)}
```